BedrockServer.cpp

```diff
@@ -369,11 +369,13 @@ void BedrockServer::sync(const SData& args,
             // Store this before we start writing to the DB, which can take a while depending on what changes were made
             // (for instance, adding an index).
             upgradeInProgress.store(true);
+            command.request.methodLine = "_upgradeDB";
```

Comment:
what is this for?

Reply:
Was to get the "requeued command" log line to report where we got stuck. Tyler asked me to take it out, so that was already my plan.